Makefile_gke

```diff
@@ -0,0 +1,242 @@
+#kubectl exec -it -n redpanda redpanda-src-0 -c redpanda -- /bin/bash -c "rpk cluster config set data_transforms_enabled true" && kubectl exec -it -n redpanda redpanda-src-0 -c redpanda -- /bin/bash -c "rpk topic create cr1"
+#kubectl patch statefulset redpanda-src -n redpanda --type='json' -p='[{op: remove, path: /spec/template/spec/containers/0/resources/limits}]'
+
+# useful:  alias internal-rpk="kubectl --namespace redpanda exec -i -t redpanda-src-0 -c redpanda -- rpk"
+.PHONY: redpanda
+redpanda: 
+    echo 'for GKE please run these steps manually until patch has been sorted'
+	#-$(HELM) repo add redpanda-data https://charts.redpanda.com 
+	#-$(HELM) repo update
+	#-$(HELM) upgrade --install redpanda-src redpanda-data/redpanda --version 5.8.3 -n redpanda --create-namespace --values redpanda/gkevalues.yaml  
+    #kubectl patch statefulset redpanda-src -n redpanda --type='json' -p='[{op: remove, path: /spec/template/spec/containers/0/resources/limits}]'
+    #-kubectl exec -it -n redpanda redpanda-src-0 -c redpanda -- /bin/bash -c "rpk cluster config set data_transforms_enabled true"
+    #-kubectl exec -it -n redpanda redpanda-src-0 -c redpanda -- /bin/bash -c "rpk topic create cr1"
+
+.PHONY: redpanda-wasm
+redpanda-wasm:
+	
+	@for dir in $(DIRS); do \
+		cd redpanda/$$dir/ && go mod tidy && rpk transform build && cd ../.. ;\
+		kubectl exec -it -n redpanda redpanda-src-0 -c redpanda -- /bin/bash -c "rpk topic create $$dir" ;\
+		kubectl exec -it -n redpanda redpanda-src-0 -c redpanda -- /bin/bash -c "mkdir -p /tmp/$$dir" ;\
+		kubectl cp redpanda/$$dir/transform.yaml redpanda/redpanda-src-0:/tmp/$$dir/. ;\
+		kubectl cp redpanda/$$dir/$$dir.wasm redpanda/redpanda-src-0:/tmp/$$dir/. ;\
+		kubectl --namespace redpanda exec -i -t redpanda-src-0 -c redpanda -- /bin/bash -c "cd /tmp/$$dir/ && rpk transform deploy" ;\
+	done
+
+
+# TODO: replace BRANCH to MAIN before PR is merged
+.PHONY: redpanda-wasm-hosted
+redpanda-wasm-hosted:	
+	@for dir in $(DIRS); do \
+		kubectl exec -it -n redpanda redpanda-src-0 -c redpanda -- /bin/bash -c "rpk topic create $$dir" ;\
+		kubectl exec -it -n redpanda redpanda-src-0 -c redpanda -- /bin/bash -c "rpk transform deploy --file https://raw.githubusercontent.com/k8sstormcenter/honeycluster/main/prebuilt/$$dir.wasm --name $$dir --input-topic=$$(sed -n -e 's/^input-topic: *//p' redpanda/$$dir/transform.yaml) --output-topic=$$dir --var language=tinygo-no-goroutines " ;\
+	done
+
+##@ Filters out the hashes in the baseline topic from the keygen topic and writes result to signal
+# To exclude further hashes from the signal, add them to the baseline topic
+.PHONY: ksqldb
+ksqldb:
+	kubectl apply -f ksqldb/ksqldb.yaml -n redpanda
+
+
+
+##@ Stops the population of the baseline topic
+.PHONY: baseline-signal
+baseline-signal:
+	kubectl exec -it -n redpanda redpanda-src-0 -c redpanda -- /bin/bash -c "rpk transform delete baseline --no-confirm && rpk topic trim-prefix signal -o @$$(date +%s) --no-confirm"
```

Comment:
I removed the ` && rpk topic trim-prefix signal -o @$$(date +%s) --no-confirm` part in my PR. The original intent was that ksqlDB lets some messages through while collecting the baseline because of the timing of messages coming in and it removes these messages when the baseline collection is stopped.

However, I think it's best to remove this part since people might not expect messages to be removed from signal when they execute this target.